core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3465,8 +3511,32 @@ private Name() {} // prevent instantiation
         "alluxio\\.worker\\.tieredstore\\.level(\\d+)\\.watermark\\.low\\.ratio"),
     ;
 
+    // puts property creators in a nested class to avoid NPE in enum static initialization
+    private static class PropertyCreators {
```

Comment:
`NestedPropertyCreators`?

Reply:
I renamed the individual creators instead given that the default creator will be used for non-nested properties as well.